iphone/Maps/Classes/CustomViews/MapViewControls/Search/Console/MWMConsole.mm

```diff
@@ -15,6 +15,9 @@ + (BOOL)performCommand:(NSString *)cmd
 
   if ([self performSound:cmd])
     return YES;
+  
+  if ([self perform3dMode:cmd])
```

Comment:
If you think that is better to do it in core - feel free to refactor :)
btw you can ask @kshalnev about this point.